### ISSUE TITLE
Usage with the Azure OpenAI API

### ISSUE THREAD

**Issue by pietz** [chrislemke/ChatFred#55]
The OpenAI services can also be used through Microsoft Azure. The API has exactly the same as with OpenAI and you can even use OpenAIs python library. You just need to adjust the cinfig in the following way:

```python
openai.api_type = "azure"
openai.api_base = os.getenv("OPENAI_API_BASE") 
openai.api_version = "2023-03-15-preview"
openai.api_key = os.getenv("OPENAI_API_KEY")
```

It would be great if I could change the configuation in ChatFreds settings.

**Comment by chrislemke** [comment]
Looks like a great idea. @pietz maybe you want to contribute to the project?

**Comment by pietz** [comment]
fair point. for now I'll write a quick workflow myself. I understand ChatFred does quite a bit more by now than I can code together in a few minutes, but I just require more control over the actual prompts. Thanks for building this and I'll report back once I have something to show.
